src/sass/partials/_layout.scss

```diff
@@ -247,6 +247,18 @@
   }
 }
 
+
+// Home - Announcement
+
+section.news .badge-text {
+    color: #000;
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 21px;
+    padding: 0px 0 0 13px;
```

Comment:
a bit weird that only one of the zeroes has the `px`

Reply:
I just copy-pasted whatever I found in the original template.
https://github.com/src-d/landing/pull/327/files#diff-9c713ebd1761433022f923462a301cf4L55

But you're right, let's fix it